src/styles/helpers/__test__/getContrastText.spec.js

```diff
@@ -0,0 +1,66 @@
+import { getContrastText } from '../color';
+import { COLOR_BRAND, COLOR_SUCCESS, COLOR_WARNING } from '../../colors';
+
+jest.mock('../color/getContrastRatio', () => background => {
+    if (background === '#fc0') {
+        return false;
+    }
+    return true;
+});
+jest.mock('../color/darken', () => color => {
+    if (color === '#01b6f5') {
+        return 'rgb(0, 163, 220)';
+    }
+    return 'rgb(26, 209, 163)';
```

Comment:
Since these values are used below you should put darkenColorBrand and darkenColorSuccess above this mock and use these constants here.